apps/docs/src/.vitepress/components/BrowsersList.vue

```diff
@@ -0,0 +1,63 @@
+<script lang="ts" setup>
+import { data as browsersData } from "#src/.vitepress/browser-loader.data";
+
+const browsers = browsersData.browsers.filter((b) => b.coverage > 0);
+
+const popularBrowsers = ["chrome", "edge", "firefox", "safari", "samsung", "ios_saf"];
+
+const filteredBrowsers = popularBrowsers
+  .map((pb) => {
+    const b = browsers.find((b) => b.id === pb);
+    if (b)
+      return {
+        ...b,
+        minVersion: b?.versions ? Object.keys(b.versions).sort()[0] : "",
+      };
+  })
+  .filter((b) => !!b);
+</script>
+
+<template>
+  <ul class="browsersList">
+    <li v-for="{ name, minVersion, id } in filteredBrowsers" :key="name" class="browser">
+      <img
+        :src="`/assets/browsers/${id}.svg`"
+        :alt="`${name} icon`"
+        width="40px"
+        height="40px"
+        class="browser__image"
+      />
+      <p class="browser__name">{{ name }}</p>
+      <p>Version ≥ {{ minVersion }}</p>
+    </li>
+  </ul>
+</template>
+
+<style lang="scss" scoped>
+.browsersList {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(160px, 1fr));
+  row-gap: var(--onyx-spacing-lg);
```

Comment:
```suggestion
  display: flex;
  flex-wrap: wrap;
  gap: var(--onyx-spacing-xl);
```

We could use flex + flex-wrap instead so we don't need to maintain the 160px width manually and instead let the components take whatever width they need :) What do you think?